cmd/fireaptos/cli/common-flag.go

```diff
@@ -23,9 +23,9 @@ import (
 func init() {
```

Comment:
Please also create a `CHANGELOG.md` at the root of the project that describe the breaking changes we are introducing here.

You can use https://github.com/streamingfast/firehose-ethereum/blob/develop/CHANGELOG.md#v100 as your base for it, some of the flags renamed are already documented in the linked document.

I put the comment here but it's a general comment unrelated to the file, I added to keep track of it.